EditorPluginExample/Resources/EditorData/Example.plugin.ts

```diff
@@ -0,0 +1,91 @@
+/// <reference path="../../typings/Atomic/AtomicWork.d.ts" />
+/// <reference path="../../typings/Atomic/EditorWork.d.ts" />
+
+const ExamplePluginUILabel = "Example Plugin";
+const ExamplePluginTBPath = "EditorData/Example.tb.txt";
+
+class ExamplePluginService implements Editor.HostExtensions.HostEditorService, Editor.HostExtensions.ProjectService {
+
+    name: string = "ExampleService";
+    description: string = "This service demonstrates plugin functionality functionality.";
+
+    private serviceLocator: Editor.HostExtensions.HostServiceLocator = null;
+    private extensionWindow: Editor.Modal.ExtensionWindow = null;
+
+    private helloLabel: Atomic.UITextField;
+    private nameField: Atomic.UIEditField;
+
+    initialize(serviceLoader: Editor.Extensions.ServiceLoader) {
```

Comment:
I'll take a look at that and see if I can clean it up.  The idea was that the ```HostServiceLocator.uiServices``` would contain methods that the extension would call to interact with the Editor ui as well as allowing the extension to register for UI events.  The ```UIService``` interface is really just a way to describe the events that should be listened to.  Maybe ```UIService``` should become ```UIServiceHandlers``` and leave HostServiceLocator.uiServices as is?

Great feedback, btw! :)